### ISSUE TITLE
Ignite as a cache for SQL Server database

### ISSUE THREAD

**Issue by zuhairamahdi** [apache/ignite#11622]
Hi all,

is there a way to configure and make apache ignite as a caching layer for SQL Server database. something similar to Redis RDI 'Only available in the enterprise version'.

**Comment by ptupitsyn** [comment]
Yes, you can use [SQL Server JDBC driver](https://learn.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server) and `CacheJdbcPojoStore` - see [RDBMS Integration](https://ignite.apache.org/docs/latest/persistence/external-storage#rdbms-integration) docs.

**Comment by zuhairamahdi** [comment]
Thanks @ptupitsyn is there any example 'docker-compose' or something similar that can kick start it? I'm not very familiar with Ignite. we are having this issue with some tables in SQL Server with millions of records with 1000s requests per minute making it very slow to only depend on SQL. that's why we want to create a cache where all the read query will be using it. and normal write to SQL that it will be 'somehow' synced to Ignite. I'm currently looking into some CDC solutions like Debezium and such to solve this issue.

**Comment by ptupitsyn** [comment]
There is a full example based on embedded H2 database:
https://github.com/apache/ignite/blob/master/examples/src/main/java/org/apache/ignite/examples/datagrid/store/jdbc/CacheJdbcStoreExample.java

Switching to SQL Server should be a matter of replacing the driver and connection string.
